Source/FieldSolver/FiniteDifferenceSolver/ApplySilverMuellerBoundary.cpp

```diff
@@ -57,23 +60,50 @@ void FiniteDifferenceSolver::ApplySilverMuellerBoundary (
         Array4<Real> const& Er = Efield[0]->array(mfi);
         Array4<Real> const& Et = Efield[1]->array(mfi);
         Array4<Real> const& Ez = Efield[2]->array(mfi);
+        Array4<Real> const& Br = Bfield[0]->array(mfi);
         Array4<Real> const& Bt = Bfield[1]->array(mfi);
         Array4<Real> const& Bz = Bfield[2]->array(mfi);
 
         // Extract tileboxes for which to loop
+        Box tbr  = mfi.tilebox(Bfield[0]->ixType().toIntVect());
         Box tbt  = mfi.tilebox(Bfield[1]->ixType().toIntVect());
         Box tbz  = mfi.tilebox(Bfield[2]->ixType().toIntVect());
 
         // We will modify the first (i.e. innermost) guard cell
         // (if it is outside of the physical domain)
         // Thus, the tileboxes here are grown by 1 guard cell
+        tbr.grow(1);
         tbt.grow(1);
         tbz.grow(1);
 
         // Loop over the cells
-        amrex::ParallelFor(tbt, tbz,
+        amrex::ParallelFor(tbr, tbt, tbz,
             [=] AMREX_GPU_DEVICE (int i, int j, int /*k*/){
 
+                // At the +z boundary (innermost guard cell)
```

Comment:
Hi Rémi, Thanks for asking me to review this PR.
Everything in the code and test looks very good to me.

Are you focusing only on the +z boundary for the moving window cases?

To help with the review, I tried to do the calculations (also to expand the PDF and update it for others to use it), but I got different equations although the coefficients all match yours.
Do you know where my deduction is different than yours?

[SilverMueller.pdf](https://github.com/ECP-WarpX/WarpX/files/6350166/SilverMueller.pdf)

Thanks